manifests/overlays/prod/obc.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: service-catalog-bucket-claim
+spec:
+  generateBucketName: service-catalog-bucket-claim-
+  storageClassName: openshift-storage.noobaa.io
+  additionalConfig:
```

Comment:
Right now these values are "forever", changing them later on will not update quota for the bucket, it's not reconciled by rook operator. I think we should rather remove it from here instead.

Reply:
Resolved in the latest commit.